package.json

```
@@ -0,0 +1,102 @@
+{
+  "name": "track-mail-2019-max",
+  "homepage": "https://your_gh_login.github.io/your_repo_name",
+  "description": "Mail.Ru Group Frontend Track course, 2019",
+  "version": "0.1.0",
+  "author": "Max",
+  "dependencies": {
+    "@emotion/core": "^10.0.14",
+    "@emotion/styled": "^10.0.14",
+    "babel-core": "^6.26.3",
+    "babel-preset-stage-0": "^6.24.1",
+    "cross-env": "^5.2.0",
+    "dotenv": "^8.0.0",
+    "node-sass": "^4.12.0",
+    "normalize.css": "^8.0.1",
+    "postcss": "^7.0.17",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-redux": "^7.1.0",
+    "react-router-dom": "^5.0.1",
+    "react-scripts": "^3.0.1",
+    "redux": "^4.0.1",
+    "redux-devtools-extension": "^2.13.2",
+    "redux-thunk": "^2.3.0",
+    "sass-loader": "^7.1.0"
+  },
+  "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/preset-react": "^7.0.0",
+    "@wemake-services/eslint-config-javascript": "^0.3.0",
+    "@wemake-services/eslint-config-jest": "^0.1.5",
+    "@wemake-services/eslint-config-jsdoc": "^0.3.0",
+    "@wemake-services/stylelint-config-scss": "^0.1.4",
+    "babel-plugin-lodash": "^3.3.4",
+    "babel-plugin-styled-components": "^1.10.6",
+    "babel-plugin-transform-decorators-legacy": "^1.3.5",
+    "check-engine": "^1.8.0",
+    "eslint-config-airbnb": "^15.1.0",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-babel": "^5.3.0",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-jsx-a11y": "^5.1.1",
+    "eslint-plugin-prettier": "^2.6.0",
+    "eslint-plugin-react": "^7.4.0",
+    "husky": "^0.14.3",
+    "gh-pages": "^2.1.1",
+    "jest": "^24.8.0",
+    "jest-transform-stub": "^2.0.0",
+    "jscpd": "^2.0.15",
+    "prettier": "^1.14.2",
+    "pretty-quick": "^1.6.0",
+    "remark-cli": "^6.0.1",
+    "remark-lint": "^6.0.5",
+    "remark-preset-lint-markdown-style-guide": "^2.1.3",
+    "rosie": "^2.0.1",
+    "stylelint": "^9.10.1",
+    "stylelint-a11y": "^1.2.0",
+    "stylelint-config-css-modules": "^1.4.0",
+    "stylelint-no-unsupported-browser-features": "^3.0.2",
+    "stylelint-webpack-plugin": "^0.10.5"
+  },
+  "browserslist": [
+    "> 5%",
+    "last 2 versions",
+    "not ie < 11",
+    "not ie_mob < 11",
+    "not dead"
+  ],
+  "scripts": {
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build",
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "eject": "react-scripts eject",
+    "format": "prettier --write '**/*.{js,jsx,json,md}'",
+    "format:changed": "pretty-quick",
+    "format:staged": "pretty-quick --staged",
+    "eslint-check": "eslint --print-config .eslintrc.js | eslint-config-prettier-check",
+    "lint": "npm run lint:js && npm run lint:dup && npm run lint:css",
+    "lint-fix": "eslint --fix",
+    "clean": "rm -rf dist",
+    "test": "npm run test:engine && npm run test:unit && npm run lint",
```

Нужно добавить `CI=true` в начало строки (это переменная окружения, которая будет устанавливаться в окружение виртуалки при сборке (в нашем случае на трэвисе)
```suggestion
    "test": "CI=true npm run test:engine && npm run test:unit && npm run lint",
```

---

Изменил package.json